### PR TITLE
ex3_js-objects-part1

### PR DESCRIPTION
**Pull request by smilya**
Solutions for tasks 01 through to 06 initial presentation for review

### CODE REVIEW

File: src/ex3_js-objects-part1/task-03.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+function isThereKeyInObj(key, obj) {
+  if (key in obj) {return true};
```
Comment (aneelia) [inline src/ex3_js-objects-part1/task-03.js]: попробуй решить в одну строку

File: src/ex3_js-objects-part1/task-03.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+function isThereKeyInObj(key, obj) {
```
Comment (aneelia) [inline src/ex3_js-objects-part1/task-03.js]: сложна.  isKeyInObj

File: src/ex3_js-objects-part1/task-04.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+// lint is disabled as it alerts initial obj modification required by the task
```
Comment (aneelia) [inline src/ex3_js-objects-part1/task-04.js]: можно через Object.assign

File: src/ex3_js-objects-part1/task-03.js (hunk 1 above)
Comment (smilya) [reply src/ex3_js-objects-part1/task-03.js]: Done. One string solution is suggested

File: src/ex3_js-objects-part1/task-03.js (hunk 2 above)
Comment (smilya) [reply src/ex3_js-objects-part1/task-03.js]: Done. The function name is shortened.

File: src/ex3_js-objects-part1/task-04.js (hunk 3 above)
Comment (smilya) [reply src/ex3_js-objects-part1/task-04.js]: Done. The initial object is modified through Object.assign method.
Lint disabling is no more needed.

File: src/ex3_js-objects-part1/task-04.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+function assignNewKey(key, obj) {
+  if(!(key in obj)) {
+    Object.assign(obj, {[key]: 'new'});
```
Comment (aneelia) [inline src/ex3_js-objects-part1/task-04.js:5]: 🔥

**Pull request merged**